lisa/sut_orchestrator/azure/features.py

```diff
@@ -345,35 +343,15 @@ def _read(self) -> str:
 
     def _get_console_log(self, saved_path: Optional[Path]) -> bytes:
         platform: AzurePlatform = self._platform  # type: ignore
-        compute_client = get_compute_client(platform)
-        with global_credential_access_lock:
-            diagnostic_data = (
-                compute_client.virtual_machines.retrieve_boot_diagnostics_data(
-                    resource_group_name=self._resource_group_name, vm_name=self._vm_name
-                )
-            )
-        if saved_path:
-            screenshot_raw_name = saved_path.joinpath("serial_console.bmp")
-            screenshot_name = saved_path.joinpath("serial_console.png")
-            screenshot_response = requests.get(
-                diagnostic_data.console_screenshot_blob_uri
-            )
-            with open(screenshot_raw_name, mode="wb") as f:
-                f.write(screenshot_response.content)
-            try:
-
-                with Image.open(screenshot_raw_name) as image:
-                    image.save(screenshot_name, "PNG", optimize=True)
-            except UnidentifiedImageError:
-                self._log.debug(
-                    "The screenshot is not generated. "
-                    "The reason may be the VM is not started."
-                )
-            unlink(screenshot_raw_name)
-
-        log_response = requests.get(diagnostic_data.serial_console_log_blob_uri)
+        log_response_content = save_console_log(
```

Comment:
Why create a variable only to return it?

```suggestion
        return  save_console_log(
```

Reply:
updated